### DISCUSSION TITLE
Primal diverging immediately

### DISCUSSION THREAD

**Discussion by arnalj** [mdolab/dafoam#468] [General]
Hello, I'm setting up a problem for aerodynamic optimization. I have the open foam case set up and working well when I run `simpleFoam`. However, when I run the same case via a defoam run script with `DAsimpleFoam`, the solution diverges immediately. What could potentially be causing this?

See the attached run script
[runScript_Aero.py.zip](https://github.com/mdolab/dafoam/files/12336877/runScript_Aero.py.zip)

**Comment by friedenhe** [comment]
It is likely that your FFD does not fully contain the design surface.

**Reply by arnalj** [reply]
I was able to get solutions by changing the turbulence model. Originally I was working with kOmegaSST. I switched to the SSA model (fv3) and it runs now. Not sure what could be the problem because the original case ran with simpleFoam
